capsul/qt_gui/widgets/pipeline_file_warning_widget.py

```diff
@@ -0,0 +1,89 @@
+##########################################################################
+# CAPSUL - Copyright (C) CEA, 2015
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+# System import
+from soma.qt_gui.qt_backend import QtCore, QtGui
+try:
+    import traits.api as traits
+except ImportError:
+    import enthought.traits.api as traits
+
+
+class PipelineFileWarningWidget(QtGui.QSplitter):
+
```

Comment:
A bit of docstring to give some hint of the usage of this class in the GUI would be a plus.